x-pack/metricbeat/tests/system/test_xpack_base.py

```diff
@@ -1,9 +0,0 @@
-import os
-
-import xpack_metricbeat
-import test_base
-
-
-class Test(xpack_metricbeat.XPackTest, test_base.Test):
```

Comment:
Oh sorry, if we remove this test no base test is executed for metricbeat. We can leave it empty as no override is needed now for kibana dir.
```
 class Test(xpack_metricbeat.XPackTest, test_base.Test):
    pass
```

Reply:
Added it back.